client/src/components/Request.js

```diff
@@ -78,6 +79,11 @@ const reducer = (state, action) => {
         loading: false,
         error: action.error,
       };
+    case "STATUS_SENT":
```

Comment:
Our local reducers are getting really big. Maybe consider moving them into 'reducers' folder

Reply:
Agreed. Will clean that up in another branch